lib/pages/chat_blank/chat_blank.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:fluffychat/pages/chat_blank/chat_blank_style.dart';
+import 'package:fluffychat/resource/image_paths.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:linagora_design_flutter/linagora_design_flutter.dart';
+
+class ChatBlank extends StatelessWidget {
+  final bool loading;
+  const ChatBlank({this.loading = false, Key? key}) : super(key: key);
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      // Add invisible appbar to make status bar on Android tablets bright.
+      appBar: AppBar(
+        automaticallyImplyLeading: false,
+        elevation: 0,
+        backgroundColor: Theme.of(context).scaffoldBackgroundColor,
+      ),
+      extendBodyBehindAppBar: true,
+      body: loading ? _buildLoading(context) : _buildNotChatWidget(context),
+    );
+  }
+
+  Widget _buildNotChatWidget(BuildContext context) {
+    return Column(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        Center(
+          child: Hero(
+            tag: 'info-logo',
+            child: SvgPicture.asset(
+              ImagePaths.icEmptyPage,
+            ),
+          ),
+        ),
+        Padding(
+          padding: ChatBlankStyle.elementsPadding,
+          child: Text(
+            L10n.of(context)!.notInAChatYet,
+            style: Theme.of(context).textTheme.headlineLarge?.copyWith(
+                  fontWeight: FontWeight.w600,
+                ),
+          ),
+        ),
+        _buildRichText(context)
+      ],
+    );
+  }
+
+  Widget _buildLoading(BuildContext context) {
+    return Center(
+      child: SizedBox(
+        width: ChatBlankStyle.width(context),
+        child: const LinearProgressIndicator(),
+      ),
+    );
+  }
+
+  RichText _buildRichText(BuildContext context) {
+    final textStyle = Theme.of(context).textTheme.bodyLarge?.copyWith(
+          fontSize: ChatBlankStyle.textFontSize,
+          fontWeight: ChatBlankStyle.textFontWeight,
+          color: LinagoraRefColors.material().tertiary[20],
+        );
+
+    return RichText(
+      textAlign: TextAlign.center,
+      text: TextSpan(
+        children: [
```

Comment:
@imGok 
```dart
children: L10n.of(context)!.blankChatTitle
  .splitMapJoin(RegExp(r'{EditIcon}'),
    onMatch: (_) => WidgetSpan(),
    onNonMatch: (text) => TextSpan(text: text);
```

Reply:
couldn't use {Icon} because it's used by L10n for interpolation already. I used #EditIcon#